ethcore/res/ethereum/expanse.json

```diff
@@ -15,11 +15,11 @@
 				"difficultyHardforkTransition": "0x59d9",
 				"difficultyHardforkBoundDivisor": "0x0200",
 				"bombDefuseTransition": "0x30d40",
-				"eip150Transition": "0x7fffffffffffffff",
-				"eip155Transition": "0x7fffffffffffffff",
-				"eip160Transition": "0x7fffffffffffffff",
-				"eip161abcTransition": "0x7fffffffffffffff",
-				"eip161dTransition": "0x7fffffffffffffff"
+				"eip150Transition": "0x927C0",
+				"eip155Transition": "0x927C0",
```

Comment:
With EIP155 enabled, you may want to specify a "chainID" in params. Otherwise, transactions from the Ethereum mainnet will still be replayable onto expanse and vice-versa as they both specify network ID 1, defeating the point a bit.